### PR TITLE
LPS-88623 Person client test

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
Perfect!

**Comment by ruben-pulido** [comment]
Looks great @dgarciasarai! 👏

File: modules/apps/headless-apio/person/person-apio-client-test/src/testIntegration/java/com/liferay/person/apio/client/test/MyUserAccountApioTest.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+	public void tearDown() {
+		ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).when(
+		).delete(
+			_userHrefURL
+		).then(
+		).statusCode(
+			200
```
Comment (ruben-pulido) [inline modules/apps/headless-apio/person/person-apio-client-test/src/testIntegration/java/com/liferay/person/apio/client/test/MyUserAccountApioTest.java]: I just remembered that we need to use `Matchers.isOneOf(200, 204)` until an APIO bug is fixed, so tests pass both in master and 7.1.x

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

File: modules/apps/headless-apio/person/person-apio-client-test/src/testIntegration/java/com/liferay/person/apio/client/test/MyUserAccountApioTest.java (hunk 1 above)
Comment (dgarciasarai) [reply modules/apps/headless-apio/person/person-apio-client-test/src/testIntegration/java/com/liferay/person/apio/client/test/MyUserAccountApioTest.java]: Thanks @ruben-pulido! 👍

**Pull request closed without merging**